pyccel/codegen/printing/fcode.py

```diff
@@ -2640,15 +2640,17 @@ def _print_Header(self, expr):
 
     def _print_SysExit(self, expr):
         code = ""
+        if isinstance(expr.status, LiteralInteger):
+            arg = str(expr.status)
         if expr.status.dtype is not NativeInteger() or expr.status.rank > 0:
             print_arg = FunctionCallArgument(expr.status)
             code = self._print(PythonPrint((print_arg, ), file="stderr"))
             arg = "1"
         else:
-            arg = expr.status
-            if arg.precision != 4:
-                arg = NumpyInt32(arg)
-            arg = self._print(arg)
+            # Ensure no type specifier is added for NativeInteger
+            arg = self._print(expr.status)
```

Comment:
I think this is one of the rare exceptions where it is not worth calling `self._print` as you then have to correct what it did. Why not just convert the integer to a string directly?

Reply:
@EmilyBourne , I have added the  change for `int` to `str`.